Automattic/wp-calypso
# Update P2 signup 'hooray' screen to new design

Addresses 4797-gh-Automattic/p2 
 
#### Changes proposed in this Pull Request

* Change the design of he P2 Workspace creation loading screen 

from:

<img width="1508" alt="image" src="https://user-images.githubusercontent.com/1574407/157052641-08bce25f-4c2f-4322-90b7-ea523df22348.png">


to:

<img width="1492" alt="image" src="https://user-images.githubusercontent.com/1574407/157052434-4f4974b7-642d-4515-aa69-b5ce2ff282ad.png">


#### Testing instructions



* Run this branch locally or via the Calypso live branch
* Navigate to `/start/p2/p2-site`
* Fill in the two inputs to create a new workspace and submit the form
* Ensure the loading screen looks as in the attached screenshots or the linked issue design files.
* Ensure the workspace name is correctly shown in the loading screen detail text.
